src/common/library/index.ts

```diff
@@ -240,6 +240,7 @@ export function buildLibrary(config: LibraryConfig) {
                     let code =
                         transformed.code
                             ?.replace(/import '\.(.*)\.scss';/g, "import '.$1.css';")
```

Comment:
may be we could just update regex?

```suggestion
                            ?.replace(/import ['"]\.(.*)\.scss['"];/g, "import '.$1.css';")
```